packages/components/select/select.component.ts

```diff
@@ -714,7 +716,9 @@ export class KbqSelect
             return;
         }
 
-        this.triggerRect = this.trigger.nativeElement.getBoundingClientRect();
+        this.triggerRect = this.parentFormField
+            ? this.parentFormField.getConnectedOverlayOrigin().nativeElement.getBoundingClientRect()
+            : this.trigger.nativeElement.getBoundingClientRect();
```

Comment:
Как вариант, чтобы дважды не вызывать `getBoundingClientRect`
```suggestion
this.triggerRect = (this.parentFormField?.getConnectedOverlayOrigin().nativeElement || this.trigger.nativeElement).getBoundingClientRect();
```

Либо
```
const originElement = this.parentFormField
    ? this.parentFormField.getConnectedOverlayOrigin().nativeElement
    : this.trigger.nativeElement;

this.triggerRect = originElement.getBoundingClientRect();
```